ui/src/components/docs/ContextDocs.vue

```diff
@@ -25,96 +44,99 @@
 </template>
 
 <script lang="ts" setup>
-    import {ref, watch, computed, getCurrentInstance, onUnmounted, nextTick} from "vue";
+    import {ref, watch, computed, getCurrentInstance, onUnmounted, onMounted, nextTick} from "vue";
     import {useStore} from "vuex";
     import {useI18n} from "vue-i18n";
-
     import OpenInNew from "vue-material-design-icons/OpenInNew.vue";
-
     import {MDCRenderer, getMDCParser} from "@kestra-io/ui-libs";
     import DocsLayout from "./DocsLayout.vue";
     import ContextDocsLink from "./ContextDocsLink.vue";
     import ContextChildCard from "./ContextChildCard.vue";
     import DocsMenu from "./ContextDocsMenu.vue";
+    import ContextDocsSearch from "./ContextDocsSearch.vue";
     import ContextInfoContent from "../ContextInfoContent.vue";
     import ContextChildTableOfContents from "./ContextChildTableOfContents.vue";
 
+    const LAST_DOC_PATH_KEY = "kestra_last_doc_path";
     const store = useStore();
     const {t} = useI18n({useScope: "global"});
 
     const docWrapper = ref<HTMLDivElement | null>(null);
+    const docHistory = ref<string[]>([]);
+    const currentHistoryIndex = ref(-1);
+    const ast = ref<any>(undefined);
 
     const pageMetadata = computed(() => store.getters["doc/pageMetadata"]);
-    const docPath = computed(() => store.getters["doc/docPath"]);
+    const docPath = computed({
+        get: () => store.getters["doc/docPath"],
+        set: (val) => {
+            store.commit("doc/setDocPath", val);
+            if (val) localStorage.setItem(LAST_DOC_PATH_KEY, val);
```

Comment:
After this change, we will not have a simple way to go back to the docs home page anymore.
For navigation, it is unsettling and upsetting for discovering users as when they navigate they can get stuck in a bad place.

Could we move this change to a different PR and add the home link there?

```suggestion
```